woocommerce/navigation
# Adding 24px spacing to 2nd menu group within navigation (#121)

Fixes #121

Adding margin between the two menu subgroups, per the design.

### Screenshots

![image](https://user-images.githubusercontent.com/444632/95525437-f96ffb80-0988-11eb-8530-96ee0f353d8a.png)


### Detailed test instructions:

- Ensure navigation plugin is active
- Click on WooCommerce to view new navigation
- Ensure the margin is present as per the screenshot
